test/distributed/fsdp/test_wrap.py

```diff
@@ -415,6 +415,46 @@ def test_auto_wrap_smoke_test(self, fsdp_init_mode, cpu_offload):
         except FileNotFoundError:
             pass
 
+    def test_always_wrap_with_ignored_modules(self):
+        sequential = TestFSDPWrap.NestedSequentialModel.get_model(cuda=False)
+        ignored_modules = [sequential[1], sequential[2][0]]
+        model = FSDP(
+            sequential,
+            process_group=self.process_group,
+            auto_wrap_policy=always_wrap_policy,
+            ignored_modules=ignored_modules,
+        )
+        # All non-ignored modules should be wrapped with FSDP
+        self.assertTrue(isinstance(model, FSDP))
+        self.assertTrue(isinstance(model.module[0], FSDP))
+        self.assertTrue(isinstance(model.module[1], nn.Linear))
+        self.assertTrue(isinstance(model.module[2], FSDP))
+        self.assertTrue(isinstance(model.module[2].module[0], nn.Linear))
```

Comment:
just confirming these are the ignored modules?

Reply:
Yes. `model.module[1]` is `sequential[1]`, and `model.module[2].module[0]` is `sequential[2][0]`, which are ignored on L420.